ibis/backends/dask/tests/test_core.py

```diff
@@ -108,58 +99,6 @@ def tmp_left_join_exe(op, lhs, **kwargs):
     assert count[0] == 1
 
 
-def test_is_computable_input():
```

Comment:
There's an exact copy of this test in the pandas backend, so this test can safely be deleted entirely.